docs/yaml-config.rst

```diff
@@ -103,7 +103,7 @@ https://goo.gl/forms/AKEoeWHixlzVfqKT2
 python.setup_py_install
 ```````````````````````
 
-* Default: `False`
+* Default: `True`
```

Comment:
Shouldn't this be done in the code?

I mean, I think it should be False by default.